docs/metrics.md

```
@@ -21,10 +21,10 @@ ocm_agent_operator_pull_secret_invalid{ocmagent_name="ocmagent"} = 0
 
 Type: Gauge
 
-Description: This gauge is set to `1` if OCM Agent Operator cannot find the `OCM Agent` custom resource, 
+Description: This gauge is set to `1` if OCM Agent Operator cannot find the `OCM Agent` custom resource,
 or `0` if it can do so successfully.
 
 Example:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)